src/components/Tweet.js

```diff
@@ -1,26 +1,33 @@
-function Tweet() {
+import ProfileImage from "./ProfileImage";
+import User from "./User"
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+
+
+function Tweet(props) {
+
+console.log(props)
+
   return (
     <div className="tweet">
-      <img
-        src="https://i.imgur.com/9yw1Fyw.jpg"
-        className="profile"
-        alt="profile"
-      />
 
+      <ProfileImage image={props.tweet.user.image} />
+      
       <div className="body">
         <div className="top">
```

Comment:
This div's should be inside the respective components. The point of having a lot of components is to have a cleaner file:
```
function Tweet(props) {
  return (
    <div className="tweet">
      <ProfileImage image={props.tweet.user.image} />
      <User userData={props.tweet.user} />
      <Timestamp time={props.tweet.timestamp} />
      <Message message={props.tweet.message} />
      <Actions />
      <i className="fas fa-ellipsis-h"></i>
    </div>
  );
}
```